src/FilterPluginManager.php

```diff
@@ -25,6 +25,9 @@
  * Enforces that filters retrieved are either callbacks or instances of
  * FilterInterface. Additionally, it registers a number of default filters
  * available, as well as aliases for them.
+ *
+ * @final
+ * @extends AbstractPluginManager<FilterInterface|callable>
```

Comment:
`|callable` is a bit too wide, since it allows for `callable(): void`

I think `callable(): mixed` or `callable(mixed): mixed` would be more appropriate